src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int i = random.nextInt(Color.values().length);
```

Comment:
Do not use abbreviations for the naming
```suggestion
        int index = random.nextInt(Color.values().length);
```